### DISCUSSION TITLE
I can't enter the hexadecimal code generated in HomeSpan into the new accessory in the mobile phone

### DISCUSSION THREAD

**Discussion by mehdizahedi1379** [HomeSpan/HomeSpan#650] [Q&A]
How can I change accessory code

**Comment by HomeSpan** [comment]
There is no hexadecimal code used in HomeKit or in HomeSpan.  There is only an 8-digit pairing Setup Code, same as with all commercial HomeKit devices.  You may be confusing the Setup Code with the Device ID, which is a hexadecimal number that is unrelated to pairing a new Accessory.  See [FAQ](https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#what-is-the-setup-code-used-for-pairing-a-homespan-device-to-homekit) for more information.
